### ISSUE TITLE
False Positive for CVE-2015-3373

### ISSUE THREAD

**Issue by stephengroat** [dependency-check/DependencyCheck#1290]
False positive on library amazon-kinesis-producer-0.12.8.jar, aws-swf-build-tools-1.1.jar, aws-swf-build-tools-1.0.jar, aws-lambda-java-core-1.1.0.jar - reported as cpe:/a:amazon_aws_project:amazon_aws:0.12.8, cpe:/a:amazon_aws_project:amazon_aws:1.1, cpe:/a:amazon_aws_project:amazon_aws:1.0, cpe:/a:amazon_aws_project:amazon_aws:1.1.0
```xml
<dependency>
   <groupId>com.amazonaws</groupId>
   <artifactId>amazon-kinesis-producer</artifactId>
   <version>0.12.8</version>
</dependency>
<dependency>
   <groupId>com.amazonaws</groupId>
   <artifactId>aws-swf-build-tools</artifactId>
   <version>1.1</version>
</dependency>
<dependency>
   <groupId>com.amazonaws</groupId>
   <artifactId>aws-swf-build-tools</artifactId>
   <version>1.0</version>
</dependency>
<dependency>
   <groupId>com.amazonaws</groupId>
   <artifactId>aws-lambda-java-core</artifactId>
   <version>1.1.0</version>
</dependency>
```

**Comment by jeremylong** [comment]
Thanks for reporting the FP!

**Comment by stephengroat** [comment]
fixed in `v3.2.1`. thanks!
